src/eclingo/parsing/transformers/function_transformer.py

```diff
@@ -0,0 +1,29 @@
+"""Module providing an AST function Trasnformer"""
+import clingo
+from clingo import ast
+from clingo.ast import Transformer
+
+
+def _rule_to_symbolic_term_adapter(rules):
```

Comment:
Is this function used anywhere besides the tests? If not, the function should go to the test file.

Reply:
Not for now, is the function I used as helper to call the transformer. It can be moved back to the tests

Reply:
This is already solved and moved to the test file. @jorgefandinno

Reply:
Ok, now I understand. It make sense to have a function that wraps the transformer. It is a good idea.
I got confused by the naming. The convention is to start by a underscore _ private functions that are only used in the module. The best idea would be to move the function back here renamed without the underscore. Prepend an underscore to the Transformer and outside of this file use only the function and never the transformer. Is that possible?

Reply:
I moved it back and removed the underscore, but I do not understand the prepend an underscore to the Transformer, should I rename the Transformer class to be _<TransformerName>? I have not seen any other Transformer being named like that

Reply:
https://github.com/potassco/python-clingox/blob/4089afed444e9e6ddea9d03092e2903680f1d748/clingox/ast.py#L815